cmd/admin_auth_ldap.go

```diff
@@ -97,6 +97,14 @@ var (
 			Name:  "avatar-attribute",
 			Usage: "The attribute of the user’s LDAP record containing the user’s avatar.",
 		},
+		cli.StringFlag{
+			Name:  "team-group-map",
+			Usage: "Map of LDAP groups to teams.",
+		},
+		cli.StringFlag{
+			Name:  "team-group-map-force",
+			Usage: "Force synchronization of mapped LDAP groups to teams.",
+		},
```

Comment:
Should this be team-group-map-removal?

Reply:
Yes, you are right, I renamed the flag and set its type to boolean (7e09991)